Clinical-Genomics/scout
# Fix Matchmaker submission things as per paper review

- Replace `MatchMaker` with `Matchmaker` in all places visible by a user
- Save diagnosis labels along with OMIM terms in Matchmaker Exchange submission objects (this is a current bug)
- Better description of HPO and OMIM terms for patients submitted to Matchmaker Exchange (current interface shows only HPO ID and OMIM term ID)

## NOTE that it might seem a lot of changes but it's mostly replacing `MatchMaker` with `Matchmaker`

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. Check the submission object for this demo case: https://scout-stage.scilifelab.se/cust101/A1383/

It looks like this:
<img width="1421" alt="image" src="https://user-images.githubusercontent.com/28093618/153184400-af9996ce-3838-4c10-b9bf-19b7ad958f01.png">

2. Install this branch on cg-vm1. Remove the patient from MME and re-add it.
3. Now the submission interface should look nicer and should show the labels for HPO and OMIM terms.

**Expected outcome**:
The functionality should be working
The interface is more compact and shows the description labels for HPO and OMIM terms:

<img width="1426" alt="image" src="https://user-images.githubusercontent.com/28093618/153561630-bb1e89fd-0e34-436e-8de0-2b1841faa65d.png">


**Review:**
